src/renderer/screens/Editor/Editor.js

```diff
@@ -76,9 +76,6 @@ const styles = theme => ({
   layerItem: {
     paddingLeft: theme.spacing.unit * 4
   },
-  layerSelect: {
```

Comment:
This entire file needs to be reverted. This is what was moving the layer picker to the right side. 

All of the changes in this file are reverting all of @algernon's changes from last week that had been agreed upon. Please see: https://github.com/keyboardio/Chrysalis/pull/389